src/test/java/codeu/model/data/HashtagTest.java

```diff
@@ -0,0 +1,50 @@
+package codeu.model.data;
+
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HashtagTest {
+
+  @Test
+  public void testCreate() {
+    String contentUpper = "Football";
+    String contentLower = "football";
```

Comment:
I think there should be a test case for an invalid tag? i.e. if my tag is "#H#a#shtag", it should differentiate the three different parts of the String containing a hashtag